docs/dev/guides/backend-routes.md

```diff
@@ -26,27 +26,39 @@ and is configured through annotations.
 // src/Controller/BackendController.php
 namespace App\Controller;
 
-use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
-use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Routing\Annotation\Route;
 use Symfony\Component\HttpFoundation\Response;
+use Twig\Environment as TwigEnvironment;
 
 /**
- * @Route("/contao", defaults={
- *     "_scope" = "backend",
- *     "_token_check" = true,
- *     "_backend_module" = "my-modules"
- * })
+ * @Route("/contao//my-backend-route",
```

Comment:
The double slash is not intended, is it?

Reply:
removed in 77a32c0